### ISSUE TITLE
investigate original "old style"  /services-disk moabs

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#70]
See https://docs.google.com/document/d/1etLnmFK07u2RfPcQcDVBNnOkq53bsdTSbbHTY2gPB38

See https://github.com/sul-dlss/shared_configs/blob/sdr-preservation-core_prod/config/environments/production.rb:

```
# /services-disk contains objects ingested in the "old style" (i.e. before versioning);
# eventually they will need to be migrated to the new style.
```

- [ ] ensure we have fixtures in this "old style" (?)
- [ ] see if methods work to get info against "old style"

**Comment by ndushay** [comment]
Note that in the handlful of fixture objects from Lynn, there *may* be some old enough to be from the initial `/services-disk`

**Comment by tallenaz** [comment]
Blocked by #118 as I understand it, because we want to use the latest version of moab-versioning in this investigation, and the latest version is only deployed to pcc-dev at present

**Comment by ndushay** [comment]
Ideas on how we can cope with the data, once it's mounted (or examined from wherever sdr-preservation-core_prod lives)

- migrate it to "new" format and/or

- determine how/if sdr_preservation_core deals with both "old" and "new" and/or
- if we can set up moab-versioning gem to speak "old" as well as "new" and/or
- if we can use a really old "moab-versioning-old" released gem pinned to an old release (but classes renamed differently so our app can use classes)

...

**Comment by ndushay** [comment]
Closing in favor of https://docs.google.com/document/d/1etLnmFK07u2RfPcQcDVBNnOkq53bsdTSbbHTY2gPB38
